### DISCUSSION TITLE
Favicon Not Displaying on Pages Despite Being Accessible from Server

### DISCUSSION THREAD

**Discussion by pokeyaro** [vercel/next.js#64919] [Help] [answered]
### Summary

Hello,

I am facing an issue with displaying the favicon on my website. I have placed the favicon.ico file in the public directory, and while I can access the favicon directly via http://localhost:3000/favicon.ico, it does not appear on any of my pages when I run my local server.
Here is the link tag I included in the <head> section of my document:
```html
<link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" sizes="16x16" />
```

Interestingly, when I move the favicon.ico file into a directory I created named static and update the <link> tag accordingly, the favicon suddenly appears on my pages without any issues:
```html
<link rel="shortcut icon" href="/static/favicon.ico" type="image/x-icon" sizes="16x16" />
```

This is quite puzzling, and I am trying to understand why this is happening.

Below are details regarding my development environment.

Can anyone provide insight into why the favicon is not visible when located in the public directory, but appears as expected when I use a custom static directory?

Thanks in advance for any help or suggestions!

### Additional information

```js
Operating System:
Platform: darwin
Arch: arm64

Binaries:
Node: 20.12.1
npm: 10.5.0
Yarn: 1.22.22
pnpm: 8.15.6

Relevant Packages:
next: 14.2.2
react: 18.2.0
```

**Comment by ananuness** [comment]
Hii @pokeyaro! It's indeed strange... Could you inform if the public folder is in the root directory?

**Reply by pokeyaro** [reply]
yes, the structure looks something like this:

```bash
project
├── prettier.config.js
├── public
├── src
│   ├── components
│   ├── images
│   ├── pages
│   └── styles
├── static
│   └── favicon.ico
└── tailwind.config.js
```

**Reply by ananuness** [reply, marked as answer]
Oh, okay! You can leave like that or if you really want it in the public folder, you have these options: 

- Do a hard refresh on the page using `ctrl + alt + r`
- Save it as `.png` instead of `.ico`
- Or inform the `href` like that and see if it works already: 

```html
<link rel=”shortcut icon” href=”%PUBLIC_URL%/favicon.ico”>
```

**Reply by pokeyaro** [reply]
Change to `.png` favicon solved this problem, it’s crazy, thx

**Reply by ananuness** [reply]
It is indeed!! I still don't know how this bug appears, but I'm glad it worked! You're welcome 💖
